wordpress-mobile/gutenberg-mobile
# [FIX] Bottom-sheet height after using back button

Fixes the issue with the height of bottom-sheet when we back from another screen inside it.

GB PR: https://github.com/WordPress/gutenberg/pull/28632

To test:
1. add button block
2. open settings
3. open the background color settings
4. select gradient segment
5. open "customize gradient"
6. use the back button on the top left of bottom-sheet
7. switch to solid segment
8. switch back to the gradient segment
9. the height should be changed

PR submission checklist:
